docs/guidance/volcano-integration.md

```diff
@@ -0,0 +1,67 @@
+# KubeRay integration with Volcano
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch scheduling system built on Kubernetes. It provides a suite of mechanisms (gang scheduling, job queues, fair scheduling policies) currently missing from Kubernetes that are commonly required by many classes of batch and elastic workloads. KubeRay's Volcano integration enables more efficient scheduling of Ray pods in multi-tenant Kubernetes environments.
+
```

Comment:
Could you also mention something to the effect of "This is a new feature. Feedback welcome."
Good to manage expectations :)